hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestSstFilteringService.java

```diff
@@ -180,13 +183,14 @@ public void testIrrelevantSstFileDeletion()
     assertThat(nonLevel0FilesCountAfterCompact).isGreaterThan(0);
 
     String bucketName2 = "buck2";
-    createVolumeAndBucket(volumeName, bucketName2);
+    addBucketToVolume(volumeName, bucketName2);
     createKeys(volumeName, bucketName2, keyCount);
 
     activeDbStore.getDb().flush(OmMetadataManagerImpl.KEY_TABLE);
     List<LiveFileMetaData> allFiles = activeDbStore.getDb().getSstFileList();
     String snapshotName1 = "snapshot1";
     writeClient.createSnapshot(volumeName, bucketName2, snapshotName1);
+    countTotalSnapshots++;
```

Comment:
I suggest extracting these two statements into a method (and using it for other similar two-liners) to:
- ensure count is incremented for each snapshot
- reduce duplication

Reply:
Done with the changes.